src/main/java/step2/domain/LottoMatchingCount.java

```diff
@@ -0,0 +1,34 @@
+package step2.domain;
+
+import java.util.Arrays;
+
+public enum LottoMatchingCount {
+    TREE(3, 5000),
+    FOUR(4, 50000),
+    FIVE(5, 1500000),
+    SIX(6, 2000000000),
```

Comment:
```suggestion
    SIX(6, 2_000_000_000),
```
위와 같이 표현해서 가독성을 높일수 있어요.

Reply:
오! 이런 방법이 있네요 적용했습니다~